meteor/client/ui/Shelf/NextInfoPanel.tsx

```diff
@@ -54,9 +54,11 @@ export class NextInfoPanelInner extends MeteorReactComponent<INextInfoPanelProps
 					}
 				)}
 			>
-				<span className="next-info-panel__name">{showAny && this.props.panel.name} </span>
-				{segmentName && <span className="next-info-panel__segment">{segmentName}</span>}
-				{partTitle && <span className="next-info-panel__part">{partTitle}</span>}
+				<div style={{ fontSize: isDashboardLayout ? 'var(--dashboard-panel-scaled-font-size)' : undefined }}>
```

Comment:
Wouldn't it make sense to move it to a class? I really don't like appending reactive styling using a CSS variable. It just mixes and mingles the CSS, JS code and JSX code everywhere. I think just using a CSS declaration here would be much preferable and cleaner.

Reply:
I did something. It might not fit the CSS class naming convention perfectly, though. Maybe all the dashboard panels should have the `dashboard-panel` class in addition to their own like `next-info-panel`?